CODEOWNERS

```diff
@@ -6,12 +6,12 @@
 /ruby/ @github/codeql-ruby
 
 # Make @xcorail (GitHub Security Lab) a code owner for experimental queries so he gets pinged when we promote a query out of experimental
```

Comment:
If we are removing @xcorail as owner of the experimental queries, can we just delete this entire section?

Reply:
I think so

Reply:
Good point
```suggestion
```